features/daemonize.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Daemonization
+    Naemon should successfully daemonize if and only if there is a valid
+    configuration available.
+
+    Scenario: Naemon fails to daemonize when an invalid system config is
+        provided
+        Given I have an invalid naemon system configuration
+        And config verification fail
+        Then naemon should fail to start
+
+    Scenario: Naemon fails to daemonize when an invalid object config is
```

Comment:
In each scenario we're testing two different types of configuration (system config and object config). We want to make sure naemon fails to initialize in both cases.